libcylinder/src/lib.rs

```diff
@@ -85,3 +91,9 @@ pub trait Context: Send {
     /// Computes the public key that corresponds to the given private key
     fn get_public_key(&self, private_key: &PrivateKey) -> Result<PublicKey, ContextError>;
 }
+
+impl<T: Context> VerifierFactory for T {
+    fn new_verifier(&self) -> Box<dyn Verifier> {
+        self.new_verifier()
```

Comment:
You should to call this explicitly as `Context::new_verifier(&self)`

Reply:
Updated